modules/local/generax.nf

```diff
@@ -22,7 +22,8 @@ process GENERAX {
     path families          // Filepath to the families file
 
     output:
-    path "*" , emit: results
+    path "*" ,                                        emit: results
+    path "GeneRax/results/*/*_reconciled_gft.newick", emit: generax_gfts
```

Comment:
You can do `path "GeneRax/results/**_reconciled_gft.newick", emit: generax_gfts` for both this and the SpeciesRax one